### PR TITLE
Fix rewards shared calculations, use current epoch data for user facing information

### PR DESCRIPTION
**Pull request by kimpers**

### CODE REVIEW

File: ts/components/staking/wizard/wizard_flow.tsx [hunk 1]
```diff
@@ -336,7 +336,7 @@ export const RecommendedPoolsStakeInputPane = (props: StakingInputPaneProps) =>
                                 key={rec.pool.poolId}
                                 name={stakingUtils.getPoolDisplayName(rec.pool)}
                                 collectedFees={rec.pool.currentEpochStats.totalProtocolFeesGeneratedInEth}
-                                rewards={1 - rec.pool.nextEpochStats.approximateStakeRatio}
```
Comment (kimpers) [inline ts/components/staking/wizard/wizard_flow.tsx:339]: This was a 🐛 right?

Comment (johnrjj) [reply ts/components/staking/wizard/wizard_flow.tsx:339]: yes! https://app.asana.com/0/1153534547690690/1155895391339009 thank you!!

Comment (johnrjj) [reply ts/components/staking/wizard/wizard_flow.tsx:339]: (I think staked amount prop is also incorrect, but I can look at that in a separate PR)

**Review by johnrjj — APPROVED** [review]

File: ts/components/staking/wizard/wizard_flow.tsx (hunk 1 above)
Comment (kimpers) [reply ts/components/staking/wizard/wizard_flow.tsx:339]: I'll fix it, no worries. It seems like my task was a duplicate of that one 😄

**Pull request merged**